### ISSUE TITLE
Auto scroll left on grid resizing/dragging 

### ISSUE THREAD

**Issue by e2raptor** [STRML/react-grid-layout#225]
I've defined a bigger number of cols and width in the RGL properties, and of course this allow me to dragg my grids beyond the window's width, but the thing is that i've to manually scroll (horizontally) to reach that grid, and on the resize side, it only allows vertical growth. I've tried different approaches but still no result. Can anyone point me a way to do this? Thanks in advance.

**Comment by STRML** [comment]
Sorry, at this point, RGL has a fixed width in columns and only allows vertical growth. Would be happy to accept a PR that tackles this but it's quite an undertaking.

**Comment by f4k3N4m3** [comment]
@e2raptor Have you found any solution (or library) for horizontal growth?

**Comment by e2raptor** [comment]
Hi @f4k3N4m3 , I finally ended up with [gridster](http://gridster.net/), which is the one I was using before, but I've tried to change to RGL since is the perfect integration for react, but one of my requirements was the horizontal growth so I had not other choice. Check this [dashboard demo](https://github.com/Zoomdata/sample-app-prettydashboard) I had to work on using React+Gridster and of course..horizontal growth. I always wanted to work on this feature for the RGL, but time hasn't been enough.
